### DISCUSSION TITLE
[Feature Request]评论区右置，方便一边看视频一边看评论

### DISCUSSION THREAD

**Discussion by SekiBetu** [the1812/Bilibili-Evolved#1870] [功能建议 / Ideas]
https://github.com/tanguykurylo/show-youtube-comments
https://chrome.google.com/webstore/detail/show-youtube-comments-whi/gonknbphdaoahjnamfjpaincammofgae
如题，这个功能有现成开源的YouTube版的可以参考

**Comment by the1812** [comment]
可以视频右置, b站自带mini播放器或者浏览器画中画都可以

**Reply by CloudyEagle** [reply]
b站自带mini播放器过小，如果可以适当增大也是可以的；
浏览器画中画无法加载弹幕，哔哩哔哩看视频没有弹幕感觉失去精髓，尽管现在的弹幕环境没有以前好。

**Comment by leonsyz** [comment]
B站的PC客户端有右侧评论区了，不过还是希望有右侧评论区插件出现

**Comment by b1tzer** [comment]
这个功能非常有用，最好和youtube一样，通过选项卡的方式可以切换评论和推荐视频

**Reply by kawatabi** [reply]
类似这种效果： https://greasyfork.org/zh-CN/scripts/501249-tabview-youtube-totara

**Comment by OverloadedCAT** [comment]
最好的方案其实还是能把播放器锁定在顶端，实现类似于手机端的上划逻辑
